OofPlugin/OofPlugin.csproj

```diff
@@ -1,15 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
-	<Import Project="Dalamud.Plugin.Bootstrap.targets" />
```

Comment:
ily, I was just doing the same. 

I think you can also delete `OofPlugin/Dalamud.Plugin.Bootstrap.targets` after this change